cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -174,3 +171,158 @@
         {{ label_text }}
     </option>
 {% endmacro %}
+
+
+{% if not value %}
+    {% do global_dict.update(
+        {
+            'filter_list_controls': {
```

Comment:
@kave @kurtw not sure how this will look on your end, but there may be a better structure for handing the form data to the filter.